tests/DCPS/Thrasher/readme.txt

```diff
@@ -0,0 +1,16 @@
+The Thrasher test is designed to test the stability and robustness of OpenDDS in scenarios that 1 to P publisher participant(s) send(s) the same number of 1 to M messages to a single subscriber participant in the same domain. Currently, P = 64, and M = 1024.
```

Comment:
```suggestion
The Thrasher test is designed to test the stability and robustness of OpenDDS in scenarios where a variable number of publisher participants send an inversely proportional number of messages to a single subscriber participant in the same domain. The threading of the test is meant to be intentionally sub-optimal in order to simulate "thrashing" (https://en.wikipedia.org/wiki/Thrashing_(computer_science)), which is where the test gets its name.
```

Reply:
A trivial observation of the original code: the number of publisher participants and the number of messages are not necessarily 'inversely proportional' (single 1:1, double 2:1, triangle 3:3, default 1:1024). If we add 'inversely proportional' to the requirements here, we need to make some changes in run_test.pl. It seems unnecessary, and without changing it here, we will have greater freedom to try different parameters later proportional or not, as long as we pass the correct expected_samples.  The concurrency issue is addressed in a separate item below.

Reply:
Well, we're in agreement that it's trivial. This is the general description of the test and not a requirement. If it really bothers you that some of the test cases don't follow this pattern, add the word "mostly" or "generally" or "usually".

Reply:
The wording is modified as suggested, removing all mentioning of P and M.